### PR TITLE
bug tracker number #0003473 - Accept z vector in Draft array tools

### PR DESCRIPTION
**Pull request by mwganson**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.

---

### CODE REVIEW

**Comment by mwganson** [comment]
https://freecadweb.org/tracker/view.php?id=3473

File: src/Mod/Draft/Draft.py [hunk 1]
```diff
@@ -1153,22 +1153,28 @@ def makeBlock(objectslist):
...
     makeArray(object,center,totalangle,totalnum,[name]) for polar array: Creates an array
     of the given object
     with, in case of rectangular array, xnum of iterations in the x direction
-    at xvector distance between iterations, and same for y direction with yvector
-    and ynum. In case of polar array, center is a vector, totalangle is the angle
-    to cover (in degrees) and totalnum is the number of objects, including the original.
-    The result is a parametric Draft Array.'''
-    if not FreeCAD.ActiveDocument:
-        FreeCAD.Console.PrintError("No active document. Aborting\n")
-        return
```
Comment (yorikvanhavre) [inline src/Mod/Draft/Draft.py]: Could you add back these 3 lines? I see no reason to remove them... Thanks!

Comment (mwganson) [reply src/Mod/Draft/Draft.py]: Done.  I don't know how that happened.  Wasn't intended.

It says branch is out of date with base branch.  Should I update branch?

Comment (yorikvanhavre) [reply src/Mod/Draft/Draft.py]: Thanks! No, no problem, I'll rebase when committing. Just waiting for the CI tests to finish.

**Comment by yorikvanhavre** [comment]
thanks! It's merged.

**Pull request closed without merging**